keel-titus-plugin/src/main/kotlin/com/netflix/spinnaker/keel/titus/verification/ContainerTestVerification.kt

```diff
@@ -0,0 +1,13 @@
+package com.netflix.spinnaker.keel.titus.verification
+
+import com.netflix.spinnaker.keel.api.Verification
+import com.netflix.spinnaker.keel.api.titus.TitusServerGroup.Location
+
+data class ContainerTestVerification(
```

Comment:
This reads backwards to me as is (as in "a container test", not "a test that runs in a container"). Suggestion:
```suggestion
data class ContainerBasedTestVerification(
```

Reply:
Changing this now